### ISSUE TITLE
SolanaExtension is not a constructor when using @magic-ext/solana in react-native

### ISSUE THREAD

**Issue by bmeeder22** [magiclabs/magic-js#265]
### ✅ Prerequisites

- [x] Did you perform a cursory search of open issues? Is this bug already reported elsewhere?
- [x] Are you running the latest SDK version?
- [x] Are you reporting to the correct repository (`magic-sdk`)?

### 🐛 Description

When attempting to use @magic-ext/solana in a build of React Native the build will error out with the following error: 
```
SolanaExtension is not a constructor
```

### 🤔 Expected behavior

When using this exact same import scheme in regular web the Magic object is able to be initialized with no errors. 

### 😮 Actual behavior

We see the following warning: 
<img width="392" alt="Screen Shot 2022-01-18 at 5 03 04 PM" src="https://user-images.githubusercontent.com/13264552/150032898-85ee5c31-2980-4aff-9e51-1948dcfd170c.png">

Screenshot of the error:
<img width="390" alt="Screen Shot 2022-01-18 at 5 05 32 PM" src="https://user-images.githubusercontent.com/13264552/150032940-54205f1d-c6ef-4d1b-b130-dcd522a1026c.png">

This is most likely due to some issue with the fact that index.native.js is referencing itself and is not importing correctly. We can see this issue happening in [this stack overflow issue](https://stackoverflow.com/questions/55664673/require-cycles-are-allowed-but-can-result-in-uninitialized-values-consider-ref)
### 💻 Code Sample

```
import {Magic} from '@magic-sdk/react-native';
import {SolanaExtension} from '@magic-ext/solana';

  const magic = new Magic('pk_live_xxxxx', {
    extensions: [
      new SolanaExtension({
        rpcUrl: 'https://api.devnet.solana.com',
      }),
    ],
  });
```
### 🌎 Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| `@magic-sdk/react-native`      |  7.0.0
| `@magic-ext/solana` | 3.1.0
| Browser          | react native
| `yarn`           | 1.22.17
| Operating System | ios

**Comment by bmeeder22** [comment]
Note:  Manually changing `export * from './index'` => `export { SolanaExtension } from './index.js';` fixes the issue

**Comment by smithki** [comment]
:rocket: Issue was released in [`@magic-ext/conflux@0.1.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/conflux%400.1.1), [`@magic-ext/solana@3.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/solana%403.0.1) :rocket:
